src/sentry/options/defaults.py

```diff
@@ -336,3 +336,6 @@
 
 # Killswitch for dropping events in symbolicate_event
 register("store.load-shed-symbolicate-event-projects", type=Any, default=[])
+
+# All Relay options (statically authenticated Relays can be registered here)
+register("relay", default={})
```

Comment:
Can we model this after the symbolicator options, where we register individual options rather than the wrapper object?

See: https://github.com/getsentry/sentry/blob/512c8f0caf0121ba07c48b9171d6aa85f1cd46c1/src/sentry/options/defaults.py#L106-L111

Reply:
ok, will register `relay.static_auth` option then, I like it more to have one top level option rather than have a key with a `.` in it.

Reply:
This just has an effect on how options are registered. In `yaml` you still write this:

```
relay:
  static_auth: true
```

Reply:
oh... ok, didn't realize that, will change